### ISSUE TITLE
Adding basic ggplot syntax at the start of the lesson

### ISSUE THREAD

**Issue by rodriguezmDNA** [swcarpentry/r-novice-gapminder-es#87]
Comparing the data visualization section between the R-ecology workshop (http://www.datacarpentry.org/R-ecology-lesson/04-visualization-ggplot2.html) and this one (https://swcarpentry.github.io/r-novice-gapminder-es/08-plot-ggplot2/index.html) I feel this lesson could benefit by starting on the structure/syntax of creating a plot plus the idea behind the grammar of graphics. 

For example, the ecology lesson provides the example of the syntax:

> `ggplot(data = <DATA>, mapping = aes(<MAPPINGS>)) +  <GEOM_FUNCTION>()`

While the spanish lesson jumps directly into the creation of a plot

> `library("ggplot2")
ggplot(data = gapminder, aes(x = gdpPercap, y = lifeExp)) +
  geom_point()`

Furthermore, I feel that it'd be beneficial to state that one of the benefits of ggplots is that we are able to add layers (ie, different "geometries" with `geom_`*) and thus create more complex plots in a more structured manner.

**Comment by raynamharris** [comment]
Thanks @rodriguezmDNA. This is a great suggestion! The SWC version doesn't explain the function well before jumping in. Would you be able to translate the beginning section of the DC lesson and replace the beginning of the SWC lesson with this new text? 

Note: I like loading only `ggplot2` rather than `tidyverse` and the first real code block should reference the gapminder data not the survey data :)

**Comment by raynamharris** [comment]
Note: if you take this on, please edit the .Rmd file, not the .md file. Thanks!

**Comment by mpru** [comment]
Hi, @rodriguezmDNA and @raynamharris. I took this on and updated the ggplot intro following your suggestions (translating the beginning section of DC, but keeping the gapminder example), but I also kept some ideas that were present in this lesson but not in DC. My worry is that following these ideas now this part will look a bit different from the original in English. I made a PR, #111 , if you think that this can go through, let me know if any edits are needed.
